app/views/posts/show.html.erb

```diff
@@ -1,9 +1,24 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>Posts</title>
-</head>
-<body>
-  <h1>Here is a list of posts for a given user</h1>
-</body>
-</html>
+<h1>Here is detail of a post for a specific user</h1>
+<div class="post-detail-row">
+  <div class="post-title">
+    <div class="title-author">
+      <span class="post-name"><%= @post.title %></span> by <span class="author-name"><%= @user.name %></span>
+    </div>
+    <div class="post-stats">
+      Comments: <%= @post.comments.count %> | Likes: <%= @post.likes.count %>
+    </div>
+  </div>
+  <div class="post-content">
+    <%= @post.text %>
+  </div>
+</div>
+
+<div class="comments-container">
+  <% @comments.each do |comment| %>
+  <%= "Comments" %>
+    <div class="comment">
+      <span class="comment-author"><%= comment.author.name %>:</span>
+      <span class="comment-text"><%= comment.text %></span>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] when I click on a post, there is an error, kindly fix
![Screenshot from 2023-11-03 16-36-29](https://github.com/arnoldnekemiah/blog/assets/66097371/8476cb4d-7007-41dc-9a8f-25a2b881be90)